test/fakes/fake_just_audio.dart

```diff
@@ -89,8 +89,7 @@ extension on Completer {
   }
 }
 
-class MockAudioPlayer implements AudioPlayerPlatform {
-  final String _id;
+class MockAudioPlayer extends AudioPlayerPlatform {
```

Comment:
Updated the fork
https://github.com/ryanheise/just_audio/blob/4ac4ab1b45bdece2146adc1f08432e775d40ec78/just_audio/test/just_audio_test.dart#L1519